internal/api/connectors/inventory/inventory_test.go

```diff
@@ -31,6 +31,8 @@ var _ = Describe("Inventory", func() {
 			Expect(*resultData.SatelliteVersion).To(Equal("6.11.3"))
 			Expect(*resultData.SatelliteOrgID).To(Equal("5"))
 			Expect(*resultData.RHCClientID).To(Equal("7bc66a39-e719-4bc5-b10a-77bfbd3a0ead"))
+			Expect(resultData.AnsibleHost).ToNot(BeNil())
+			Expect(*resultData.AnsibleHost).To(Equal("host1.mydomain.com"))
```

Comment:
This test is going to fail because the `resultData.AnsibleHost` field will be nil.  In order to resolve this, you will need to add the ansible_host field to the mocked data in line 17. 

```
+                {StatusCode: 200, Body: `{"results":[{"id":"1234","display_name":"test","facts":[{"namespace":"satellite", "facts":{"satellite_version": "6.11.3","satellite_instance_id":"5678","organization_id":"5"}}],"fqdn":"test_host", "ansible_host":"host1.mydomain.com"}]}`}
```